examples/dpo.py

```diff
@@ -145,6 +145,13 @@ def split_prompt_and_responses(sample) -> Dict[str, str]:
         report_to=script_args.report_to,
     )
 
+    # Trainer to match original paper
+    warmup_steps = 150
```

Comment:
Should we make this configurable by the user?

Reply:
Makes sense to me

Reply:
fixed